cert/selfsigned.go

```diff
@@ -32,8 +32,9 @@ var (
 )
 
 // ipAddresses returns the parserd IP addresses to use when creating the TLS
-// certificate.
```

Comment:
`lnd/cert` is its own go module, so you need to run `go mod tidy` in `lnd/cert` to add the testify dependency. 

@guggero I remember we ran into some ordering [issues](https://github.com/lightningnetwork/lnd/pull/4295) last time we updated `queue`? I think the order would be to bump the version in `lnd/go.mod` then tag `lnd/cert` on merge?